cloud.foundry.cli/src/test/java/cloud/foundry/cli/operations/ApplicationsOperationsTest.java

```diff
@@ -223,6 +216,47 @@ public void testRemoveApplicationShouldThrowNullPointerExceptionWhenApplicationN
         assertThrows(NullPointerException.class, () -> applicationsOperations.remove(null));
     }
 
+    @Test
+    public void testRenameOnCurrentNameNullThrowsNullPointerException() {
+        //given
+        ApplicationsOperations applicationsOperations = new ApplicationsOperations(
+                mock(DefaultCloudFoundryOperations.class));
+
+        //when
```

Comment:
its: "// then - when"

Reply:
I think we more often wrote `then + when` in similar comments of other test cases

Reply:
done